openpower/package/hostboot/p9Patches/hostboot-0001-Increase-uart-delay.patch

```diff
@@ -0,0 +1,27 @@
+From 0c69e87e86e62253bc5a33cadf41e3d5f80971f0 Mon Sep 17 00:00:00 2001
+From: Brian Silver <bsilver@us.ibm.com>
+Date: Wed, 5 Nov 2014 14:16:36 -0600
+Subject: [PATCH] Changes for Habanero bringup, uart delay and centaur/vddr
```

Comment:
Why don't we just symlink the pitches from the p8 directory?

This patch should never be merged because it is t the right way to do this. The serial port code should be reworked to properly use the character queue instead of doing one char at a time.